src/modules/movieList.js

```diff
@@ -0,0 +1,19 @@
+import getMovie from './apiCall.js';
+
+const createList = async () => {
+  const movieArr = [];
+  const movieList = await getMovie();
+  movieList.forEach((movie) => {
+    const newArr = {
+      id: movie.id,
+      name: movie.name,
+      moviePoster: movie.image.original,
+      altTitle: movie.name,
+      description: movie.summary,
+    };
```

Comment:
**[Optional]😊** well done👏👏 but I think it would be nice to restructure the **movieArra** instead of creating **newArr** array.